### ISSUE TITLE
Hyperlinks in WinUtil don't work

### ISSUE THREAD

**Issue by CatsArmy** [ChrisTitusTech/winutil#2500]
## Describe the bug
Cant open any links from WinUtil.
e.g.: the install page any app with a [app name] [(?)](). when clicking on the [(?)](?) doesn't open any webpage nor let me copy the hyper link, same thing with the GitHub link in the about page i click it nothing happens and i cant copy it.

## To Reproduce
Click on any of the links in WinUtil 

## Expected behavior
Opens the link via the selected default app for that link/file

## Additional context
Unsure if this is only happening to me.
Current Browser: Arc (The Browser Company).
Current OS: Windows 11 Insider Dev Channel.

psst also maybe add [Fluent Search](https://fluentsearch.net/) to the app list its a cool app just like flow launcher but with more feature but it ain't getting enough attention in my opinion. (note the website atm seems to be having a error 403 so here's the [GitHub](https://github.com/adirh3/Fluent-Search) instead if your interested.

**Comment by YusufKhalifadev** [comment]
Is Arc browser set as default browser in settings

**Comment by CatsArmy** [comment]
Of course

**Comment by YusufKhalifadev** [comment]
I don't know did you try with other browsers cause everything seems to be working fine

**Comment by CatsArmy** [comment]
Ima try it rn hold on also btw does it work for you ?

**Comment by YusufKhalifadev** [comment]
Yes it does work I'm using thorium and edge any chromium-based browsers

**Comment by CatsArmy** [comment]
Works with chrome ok so it's a arc issue

**Comment by CatsArmy** [comment]
> Yes it does work I'm using thorium and edge any chromium-based browsers

weird Arc is chromium based eh ill report it to them not a WinUtil issue it seems
